.github/workflows/cli-scripts-deploy-safe-rollout-online-endpoints.yml

```diff
@@ -10,6 +10,9 @@ on:
       - cli/deploy-safe-rollout-online-endpoints.sh
       - .github/workflows/cli-scripts-deploy-safe-rollout-online-endpoints.yml
       - cli/setup.sh
+      - cli/endpoints/online/managed
```

Comment:
Needs to be this: endpoints/online/managed/sample
The above path lots of stuff not needed by this script